apps/ledger-live-desktop/src/renderer/screens/exchange/SwapLiveApp/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import Card from "~/renderer/components/Box/Card";
+import WebPlatformPlayer from "~/renderer/components/WebPlatformPlayer";
+import { useDeepLinkListener } from "~/renderer/screens/earn/useDeepLinkListener";
+import { useLiveAppParameters } from "~/renderer/hooks/useLiveAppParameters";
+import { useLiveAppManifest } from "~/renderer/hooks/useLiveAppManifest";
+import { useLiveAppTopBarConfig } from "~/renderer/hooks/useLiveAppTopBarConfig";
+
+const DEFAULT_SWAP_APP_ID = "swap";
+
+const Swap = () => {
+  const manifest = useLiveAppManifest(DEFAULT_SWAP_APP_ID);
+  const topBarConfig = useLiveAppTopBarConfig();
+  const liveAppParameters = useLiveAppParameters();
+
+  useDeepLinkListener();
+
+  return (
+    <Card grow style={{ overflow: "hidden" }} data-test-id="swap-app-container">
+      {manifest ? (
+        <WebPlatformPlayer
```

Comment:
Not sure if we can use WebPlatformPlayer as we must avoid have the topBar itself in prod while we are doing the migration.

Reply:
Added a property `topBarDisabled` to control top bar visibility in `WebPlatformPlayer`. Checks and return `enablePlatformDevTools` value if it is `true`